src/test/tests_common.h

```diff
@@ -45,6 +45,9 @@
 
 #include "crt_internal.h"
 
+#define NUM_ATTACH_RETRIES 20
+#define DO_ASSERT 0
```

Comment:
So i see we are adding more things based on which common file operates. for example we have opts, and now we have things like NUM_ATTACH_RETRIES and DO_ASSERt.

I think we need to have something like tc_init([options]) which would setup all those things per each test. We already had situations where we forgot to set 'is_server' on some tests, and with addition of new options it adds chance for more errors